src/commands/BootstrapCommand.js

```diff
@@ -49,24 +50,25 @@ export default class BootstrapCommand extends Command {
       const linkSrc = dependency.location;
       const linkDest = path.join(pkg.nodeModulesLocation, dependency.name);
 
-      this.createLinkedDependency(linkSrc, linkDest, dependency.name, done);
+      this.createLinkedDependency(linkSrc, linkDest, dependency.name, (err) => {
+        const bin = dependency._package.bin;
```

Comment:
This should check `err` at the beginning and call `done(err)` if it exists and stop